en-US/focus-ios.xliff

```diff
@@ -351,6 +361,11 @@
         <target>Search for %@</target>
         <note>Label displayed for search button when typing in the URL bar</note>
       </trans-unit>
+      <trans-unit id="URL.pasteAndGo">
+        <source>Paste & Go/source>
```

Comment:
File is broken (unclosed source tag), & should be &amp;
https://github.com/mozilla-l10n/firefoxios-l10n/blob/master/en-US/firefox-ios.xliff#L190

Turns out this was generated manually.